overlord/devicestate/handlers_install.go

```diff
@@ -72,20 +80,30 @@ func (m *DeviceManager) doSetupRunSystem(t *state.Task, _ *tomb.Tomb) error {
 		return err
 	}
 	if useEncryption {
+		fdeDir := "var/lib/snapd/device/fde"
 		args = append(args,
 			// enable data encryption
 			"--encrypt",
 			// location to store the keyfile
-			"--key-file", filepath.Join(boot.InitramfsUbuntuBootDir, "ubuntu-data.keyfile.unsealed"),
+			"--key-file", filepath.Join(boot.InitramfsEncryptionKeyDir, "ubuntu-data.keyfile.sealed"),
+			// location to store the recovery keyfile
+			"--recovery-key-file", filepath.Join(boot.InitramfsWritableDir, fdeDir, "recovery-key"),
```

Comment:
recovery.key ?

Reply:
Yes, that would be consistent with the naming used for encryption key.